dask/dataframe/tests/test_dataframe.py

```diff
@@ -3740,3 +3739,23 @@ def test_dtype_cast():
     assert ddf.B.dtype == np.int64
     # fails
     assert ddf.A.dtype == np.int32
+
+
+@pytest.mark.parametrize("base_npart", [1, 4])
+@pytest.mark.parametrize("map_npart", [1, 3])
+@pytest.mark.parametrize("sorted_index", [False, True])
+def test_series_map(base_npart, map_npart, sorted_index):
+    np.random.seed(0)
```

Comment:
Is this necessary? I'd be concerned if the test only passed for specific rngs :)

Reply:
Haha, good point. I usually include for deterministic hashing in the case someone extends tests based on values produced by "generators". Works just fine in this case without, so will remove.